src/gridstack.ts

```diff
@@ -346,8 +346,6 @@ export class GridStack {
     opts = Utils.defaults(opts, defaults);
     this._initMargin(); // part of settings defaults...
 
-    // Now check if we're loading into 1 column mode FIRST so we don't do un-necessary work (like cellHeight = width / 12 then go 1 column)
-    this.checkDynamicColumn();

```

Comment:
this removes optimizations so this isn't the right fix even if it fixes the issue...